package.json

```diff
@@ -31,7 +31,7 @@
     "connect-fonts-firasans": "0.0.4",
     "connect-restreamer": "1.0.0",
     "consolidate": "0.10.0",
-    "convict": "0.6.0",
+    "convict": "0.8.0",
```

Comment:
@TDA make sure you are using npm 2.4+ , I suggest npm 2.10.

remove the `npm-shinkwrap.json`
and run `npm shrinkwrap` to create the new file